js/livecoin.js

```diff
@@ -361,10 +363,19 @@ module.exports = class livecoin extends Exchange {
             throw new ExchangeError (this.id + ' cancelOrder requires a symbol argument');
         await this.loadMarkets ();
         let market = this.market (symbol);
-        return await this.privatePostExchangeCancellimit (this.extend ({
+        let currencyPair = market['id'];
+        let response = await this.privatePostExchangeCancellimit (this.extend ({
             'orderId': id,
-            'currencyPair': market['id'],
+            'currencyPair': currencyPair,
         }, params));
+        let success = response['success'];
```

Comment:
This will fail in Python if there's no `success` key present in `response`...

Reply:
@kroitor , `success` should be always in place. If it's not, we need to crash.
https://www.livecoin.net/api/common